src/TransformerGo.cs

```diff
@@ -47,6 +47,14 @@ private static void TransformPropertyTypes(CodeModelGo cmg)
                     {
                         property.Name = property.ModelType.HasInterface() ? property.ModelType.GetInterfaceName() : property.ModelType.Name.Value;
                     }
+                    if (property.ModelType is SequenceTypeGo stg)

```

Comment:
did we figure out why those are missing ?

Reply:
yes it's a bug in autorest.common.  I discussed with @fearthecowboy and he suggested this workaround.